lib/kitchen/driver/proxy.rb

```diff
@@ -42,29 +42,36 @@ class Proxy < Kitchen::Driver::SSHBase
       def create(state)
         # TODO: Once this isn't using SSHBase, it should call `super` to support pre_create_command.
```

Comment:
Any thoughts on this comment? This PR moves off of SSHBase, so should a `super` be added?